CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact project, newest changes on the top.
 
+## [unreleased]
+
+### Fixed
+
+- `ui/Touchable` to invoke event callbacks updated during gesture
+
```

Comment:
This should go into the `packages/ui` CHANGELOG instead. But the integrator could easily move this there before merging.

Reply:
Good catch!